src/mlpack/core/dists/gaussian_distribution.hpp

```diff
@@ -166,12 +174,12 @@ class GaussianDistribution
 };
 
 /**
-* Calculates the multivariate Gaussian log probability density function for each
-* data point (column) in the given matrix
-*
-* @param x List of observations.
-* @param probabilities Output log probabilities for each input observation.
-*/
+ * Calculates the multivariate Gaussian log probability density function for each
```

Comment:
Capitalize the `log` here please.

Reply:
@rcurtin @ShikharJ I have added all the recommendations. Please update me if any changes are required.